common/story_cycles/story_cycle_animal_companion.txt

```diff
@@ -0,0 +1,279 @@
+﻿story_cycle_animal_companion = {
```

Comment:
The point is you shouldn't copy the vanilla. You should change the vanilla to work for both dogs, bears, ravens, etc.

You may ask why?
First, you save space.
Second, when Paradox is going to change something in these events, we won't need to move these changes to our files too.